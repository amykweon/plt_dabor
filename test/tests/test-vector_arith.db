vector x;
vector k;
vector z;
int y;

y = 2;
x = <1, 1>;
print_vector (x);

x * y;
print_vector (x);

x = x * y;
print_vector (x);

print_vector (<2, 2> * y);

x = <1, 1>;
k = <1, 2>;
z = x + k;
print_vector (z);