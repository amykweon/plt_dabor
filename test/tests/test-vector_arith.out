Hori (1)
Hori (2)
Hori (4)